### PR TITLE
Adds the most basic informations about loaders

### PR DESCRIPTION
**Pull request by michaelKaefer**
I missed this basic informations so I added it :)

### CODE REVIEW

File: components/config/resources.rst [hunk 1]
```diff
@@ -11,6 +11,10 @@ Loading Resources
     parameters to string values. To set parameters to other data types
     (e.g. boolean, integer, etc), the other loaders are recommended.
 
+Loaders can be used to make configuration parameters available to the application.
+The Dependency Injection Component provides a couple of loaders to load configuration
```
Comment (ricknox) [inline components/config/resources.rst]: My suggestion is to link to the component:

```suggestion
The :doc:`Dependency Injection Component </components/dependency_injection> provides a couple of loaders to load configuration 
```

Comment (ricknox) [inline components/config/resources.rst]: My suggestion is to link to the component:

```suggestion
The :doc:`Dependency Injection Component </components/dependency_injection>` provides a couple of loaders to load configuration 
```

**Review by ghost — CHANGES_REQUESTED** [review]

**Review by ghost — APPROVED** [review]

**Comment by OskarStark** [comment]
Note for the merges, this must go in `3.4` branch

File: components/config/resources.rst [hunk 2]
```diff
@@ -11,6 +11,10 @@ Loading Resources
     parameters to string values. To set parameters to other data types
     (e.g. boolean, integer, etc), the other loaders are recommended.
 
+Loaders can be used to make configuration parameters available to the application.
+The :doc:`Dependency Injection Component </components/dependency_injection>` provides a couple of loaders to load configuration
```
Comment (xabbuh) [inline components/config/resources.rst]: This document is about resources in the Config component. I am not sure I understand why add a sentence about the DependencyInjection component here.

Comment (michaelKaefer) [reply components/config/resources.rst]: For me the important part is not mentioning the DI component but defining the purpose of a loader (that is: making the configuration available to the application). When I first read the document it really took me a while to understand this, it was absolutely not clear to me evern after reading the document 2 or 3 times. The mentioning of the DI component maybe is wrong here but I personally find it very helpful to understand the basics.

Comment (OskarStark) [reply components/config/resources.rst]: Thank you for your feedback Michael, can we maybe add a link or a short mention here and link to the DI docs where we can locate your text? Would it make sense and fix your initial problem?

Comment (michaelKaefer) [reply components/config/resources.rst]: My problem is best solved with what I committed :)

As I said: for me the important part is defining the purpose of a loader, that is: making the configuration available to the application.

Yes, please edit may commit to fit in here (or tell me how I can do it).

Comment (xabbuh) [reply components/config/resources.rst]: > Loaders populate the configuration from different sources. The Config component defines the interface for such loaders. The DependencyInjection and Routing components come with specialized loaders for different file formats.

What do you think of something like this?

Comment (michaelKaefer) [reply components/config/resources.rst]: @xabbuh Cool, this seems perfect to me, very clear. For outsiders like me you maybe could add "... the **application's** configuration from different sources **like YAML files**." if that's correct.

Comment (xabbuh) [reply components/config/resources.rst]: Sounds good to me. Would you like to update here yourself or should we do that while merging?

Comment (michaelKaefer) [reply components/config/resources.rst]: Done, thanks for the help!

**Review by xabbuh — APPROVED** [review]

**Review by OskarStark — APPROVED** [review]

**Comment by xabbuh** [comment]
Thank you @michaelKaefer.

**Pull request merged**